src/test/java/systemtests/ViewStatusCommandSystemTest.java

```diff
@@ -0,0 +1,69 @@
+package systemtests;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.commons.core.Messages.MESSAGE_RIDES_LISTED_OVERVIEW;
+import static seedu.address.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
+
+import org.junit.Test;
+
+import seedu.address.logic.commands.ViewStatusCommand;
+import seedu.address.model.Model;
+
+public class ViewStatusCommandSystemTest extends AddressBookSystemTest {
+
+    @Test
+    public void viewStatus() {
+        String command;
+        String invalid = String.format(MESSAGE_INVALID_COMMAND_FORMAT, ViewStatusCommand.MESSAGE_USAGE);
+
+        /* Case:  lower case command word, mixed case argument -> success */
+        command = "viewstatus oPen";
+        assertCommandSuccess(command, getModel());
```

Comment:
Since this is success, consider the suggestion above for getting the enum value from a string.